extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalFooter_index.php

```diff
@@ -1,4 +1,4 @@
-<footer class="wds-global-footer <?= isset( $model['international_header'] ) ? 'wds-is-international' : 'wds-is-en' ?>">
+<footer class="wds-global-footer<?= isset ( $model['header'] ) ? '' : ( isset( $model['international_header'] ) ? ' wds-is-international' : ' wds-is-en' ); ?>">
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "international_header" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "header" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)